src/SFA.DAS.Roatp.CourseManagement.Jobs/Infrastructure/Tokens/BearerTokenGenerator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Azure.Services.AppAuthentication;
+
+namespace SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.Tokens
+{
+    public static class BearerTokenGenerator
+    {
+        private const string JwtBearerScheme = "Bearer";
+
+        public static async Task<AuthenticationHeaderValue> GenerateTokenAsync(string identifier)
```

Comment:
Where is this being used?

Reply:
Inside the startup.cs to build both HttpClients (one for das-course-api, one for das-roatp-api)